install/helm/open-match/templates/_helpers.tpl

```diff
@@ -45,7 +45,7 @@ imagePullPolicy: {{ .Values.image.pullPolicy }}
 resources:
   requests:
     memory: 100Mi
-    cpu: 100m
+    cpu: 50m
```

Comment:
This seems like a very small amount of cpu.  How did you arrive at this number?

Reply:
This only matter at cluster startup time. The process can still auto request more resource if it needs more quota.

See this link for more details https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/

Reply:
Still... where did the number come from?

Reply:
The idea is to use the minimum resource when setting up the environment. I was tempted to set it to 1m actually. 50m is a number that allows users having a functional Open Match deployed given GKE's default pool size with n1-standard-1 instance type.